pt-BR/1-5.md

```diff
@@ -0,0 +1,86 @@
+# Carregando imagens com UIImage
+
+Neste ponto temos nosso table view controller original cheio de imagens para selecionar, além de um detail view controller em nosso storyboard. Nosso próximo objetivo é mostrar a tela de detalhes quando uma linha da tabela for selecionada e, após a seleção, mostrar a imagem sem tela cheia.
+
+Para fazer isto funcionar temos que adicionar um outro método especial ao nosso `ViewController`. Este método tem o nome de `tableView(_, didSelectRowAt:)`, que receberá um valor `IndexPath` como no método `cellForRowAt`, para nos indicar qual linha foi selecionada. Agora iremos ter um pouco mais de trabalho:
+
+1. Teremos que criar uma propriedade em `DetailsViewController` que irá armazenar o nome da imagem a ser carregada.
+2. Iremos implementar o método `didSelectRowAt` de maneira que ele carregue o `DetailViewController` a partir do nosso storyboard.
+3. Finalmente, no método `ViewDidLoad()` do `DetailViewController` iremos carregar a imagem a partir do nome que definimos anteriormente e adicioná-la à ImageView.
+
+Vamos tratar cada uma destas etapas por vez, iniciando com a primeira: criar uma propriedade no `DetailViewController` para armazenar o nome da imagem a ser carregada.
+
+Esta propriedade será uma string - o nome da imagem a ser carregada - mas ela precisa ser uma string *opcional* porque quando o view controller for criado inicialmente ele não saberá o valor da propriedade. Logo após a criação ela será preenchida, mas ainda assim, quando o view controller for inicializado ela estará vazia.
+
+Então, adicione esta propriedade ao `DetailViewController`, logo após o `@IBOutlet`:
+
+    var selectedImage: String?
+
+A primeira tarefa foi realizada, vamos para a próxima: implementar `didSelectRowAt` de maneira a carregar o `DetailViewController` a partir do storyboard.
+
+Quando nós criamos o view controller de detalhes, você preencheu o storyboardID "Detail", o que nos permite carregar ele a partir do storyboard usando um método com o nome `instantiateViewController(withIdentifier:)`. Todos view controllers possuem uma propriedade chamada `storyboard` que faz referência ao storyboard pelo qual o view controller foi instanciado, ou nil. No caso do `ViewController` esta referência será ao Main.storyboard, que é o mesmo storyboard que contém o detail view controller. Por isto, vamos usar esta referência para chamá-lo.
```

Comment:
typo: Todos os view controllers instead Todos view controllers. Not sure

Reply:
I think you're right!